src/emulator/modules/SceGxm/SceGxm.cpp

```diff
@@ -32,20 +35,26 @@ using namespace glbinding;
 
 #define GXM_PROFILE(name) MICROPROFILE_SCOPEI("GXM", name, MP_BLUE)
 
-// https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function#FNV-1a_hash
-static uint64_t fnv1a(const void *data, size_t size) {
+static void sha256(char (&dst)[65], const void *data, size_t size) {
```

Comment:
Not important yet, but I'd abstract crypto stuff to a separate module.

Reply:
Looks like I may have to anyway, to work around the Windows typedef clash.